### PR TITLE
Adds notes for 4.11.6 and Telco RAN releases

### PR DESCRIPTION
**Pull request by opayne1**
Adds notes for 4.11.6 and Telco RAN

Version(s):
4.11 only

Issue:
[OSDOCS-4179](https://issues.redhat.com/browse/OSDOCS-4179)

Link to docs preview:
https://50803--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-11-release-notes.html#ocp-4-11-6

Additional information:
Links will not work yet.

### CODE REVIEW

**Comment by opayne1** [comment]
Removed telco RAN release notes. They have been added to #50930 which will merge directly after this PR.

File: release_notes/ocp-4-11-release-notes.adoc [hunk 1]
```diff
@@ -2614,7 +2614,7 @@ To update an existing {product-title} 4.11 cluster to this latest release, see x
 
 Issued: 2022-09-20
 
-{product-title} release 4.11.5, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2022:6536[RHSA-2022:6536] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2022:6535[RHBA-2022:6535] advisory.
+{product-title} release 4.11.5, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2022:6536[RHSA-2022:6536] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2022:6535[RHSA-2022:6535] advisory.
```
Comment (opayne1) [inline release_notes/ocp-4-11-release-notes.adoc:2617]: This is just a small typo I noticed.

File: release_notes/ocp-4-11-release-notes.adoc [hunk 2]
```diff
@@ -2636,4 +2636,23 @@ $ oc adm release info 4.11.5 --pullspecs
...
 ==== Updating
 
+To update an existing {product-title} 4.11 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-11-6"]
+=== RHBA-2022:6659 - {product-title} 4.11.6 bug fix update
+
+Issued: 2022-09-27
+
+{product-title} release 4.11.6 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2022:6536[RHBA-2022:6659] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:6658[RHBA-2022:6658] advisory.
```
Comment (stevsmit) [inline release_notes/ocp-4-11-release-notes.adoc:2646]: Looks like 6536 needs to be 6659.

Comment (opayne1) [reply release_notes/ocp-4-11-release-notes.adoc:2646]: Thank you!

**Pull request merged**